except_star.md

```diff
@@ -1,33 +1,33 @@
-### Disclaimer
+# Introducing try..except* syntax
 
-* I'm going to be using the `ExceptionGroup` name in this issue, even
-  though there are other alternatives, e.g. `AggregateException`. Naming of the
-  "exception group" object is outside of the scope of this issue.
+## Disclaimer
 
-* This issue is primarily focused on discussing the new syntax modification
-  proposal for the `try..except` construct, shortly called "except*".
+* We use the `ExceptionGroup` name herein, even though there
+  are other alternatives, e.g. `AggregateException`. Naming of the
+  "exception group" object is out of scope of this proposal.
 
-* I use the term "naked" exception for regular Python exceptions
-  **not wrapped** in an ExceptionGroup. E.g. a regular `ValueError`
+* We use the term "naked" exception for regular Python exceptions
+  **not wrapped** in an `ExceptionGroup`. E.g. a regular `ValueError`
   propagating through the stack is "naked".
 
-* I assume that `ExceptionGroup` would be an iterable object.
+* We assume that `ExceptionGroup` would be an iterable object.
   E.g. `list(ExceptionGroup(ValueError('a'), TypeError('b')))` would be
   equal to `[ValueError('a'), TypeError('b')]`
 
-* I assume that `ExceptionGroup` won't be an indexable object; essentially
+* We assume that `ExceptionGroup` won't be an indexable object; essentially
   it's similar to Python `set`. The motivation for this is that exceptions
   can occur in random order, and letting users write `group[0]` to access the
-  "first" error is error prone. The actual implementation of `ExceptionGroup`
-  will likely use an ordered list of errors though.
+  "first" error is error prone. Although the actual implementation of
+  `ExceptionGroup` will likely use an ordered list of errors to preserve
+  the actual occurrence order for rendering.
 
-* I assume that `ExceptionGroup` will be a subclass of `BaseException`,
+* We assume that `ExceptionGroup` will be a subclass of `BaseException`,
   which means it's assignable to `Exception.__context__` and can be
-  directly handled with `except ExceptionGroup`.
+  directly handled with `try: ... except ExceptionGroup: ...`.
 
-* The behavior of good and old regular `try..except` will not be modified.
+* The behavior of the regular `try..except` block will not be modified.
```

Comment:
```suggestion
* The behavior of the regular `try..except` statement will not be modified.
```
(Or "except block".)

FWIW I don't like writing "`except` block" -- let's just call it an "except block". The whole statement is called a "try statement". (Although we do need a way to talk about an "except * block".)

Reply:
We could call the whole statement a "try* statement" but that would likely be confusing. So I call it "try..except* statement"